kotest-framework/kotest-framework-engine/src/commonMain/kotlin/io/kotest/engine/interceptors/DumpConfigInterceptor.kt

```diff
@@ -23,6 +23,7 @@ internal object DumpConfigInterceptor : EngineInterceptor {
       return execute(context)
    }
 
-   private fun syspropEnabled() =
-      syspropOrEnv(KotestEngineProperties.dumpConfig) == true
+   fun syspropEnabled() =
+      syspropOrEnv(KotestEngineProperties.dumpConfig).toBoolean()
```

Comment:
the function will now return only true if the system property is set to true  , or false when it is set to false or not set at all